### PR TITLE
changed styles according design

### PR DESCRIPTION
**Pull request by KatePolyakov**
Screenshot 2024-06-12 14 57 48

1. Full wrapper size - fixed;
2. Change margin-top  - fixed;
3. Change padding (32 into 48px)  - fixed;
4. Change word optional from Uppercase to small case and change colour - fixed;
5. Change Input width from 185 to 209px  - fixed;

Screenshot 2024-06-12 15 09 26

### CODE REVIEW

File: frontend/src/pages/Register/Register.module.css [hunk 1]
```diff
@@ -1,66 +1,55 @@
...
-  opacity: 0px;
+  margin-bottom: 24px;
   color: var(--fg-Primary, #081821);
 }
-.TextInputField{
-  Width: 442px;
-  height: 79px;
-  gap:24px
 
+.EmailInput {
```
Comment (AnnaSolovykh) [inline frontend/src/pages/Register/Register.module.css:28]: Classnames should be in the camelCase, not in PascalCase

File: frontend/src/components/form/FormInput.module.css [hunk 2]
```diff
@@ -26,59 +21,68 @@
...
-  color: #DA3633;
+  color: #da3633;
   font-family: Roboto;
   font-size: 10px;
   font-weight: 600;
   line-height: 14px;
 }
 .errorFeedback svg {
-  margin-right: 5px ;
+  margin-right: 5px;
```
Comment (AnnaSolovykh) [inline frontend/src/components/form/FormInput.module.css:31]:
```suggestion
  margin-right: 4px;
```

File: frontend/src/pages/Register/Register.module.css [hunk 3]
```diff
@@ -1,66 +1,55 @@
...
-.formContainer {
-  width: 442px;
-  height: 678px;
-position: relative;
-  top: 56px;
-  /* left: 234px; */
-  gap: 56px;
+
+.page__wrapper {
+  padding: 56px 234px 104px;
```
Comment (AnnaSolovykh) [inline frontend/src/pages/Register/Register.module.css:14]: Great choice following BEM conventions!

**Review by AnnaSolovykh — COMMENTED** [review]
Thank you for the amazing! job Everything looks great, right according to the Figma design! However I have noticed a couple of errors in the console. Could you have a look and fix it? 
Screenshot 2024-06-12 at 18 45 28

**Review by nadezdatsygankova — COMMENTED** [review]
Great job, Kate! Thank you for your work! 👍 
Thank you for changing the class names.

**Comment by nadezdatsygankova** [comment]
> Thank you for the amazing! job Everything looks great, right according to the Figma design! However I have noticed a couple of errors in the console. Could you have a look and fix it? Screenshot 2024-06-12 at 18 45 28

I've fixed this console label. Could you please check it?

**Comment by AnnaSolovykh** [comment]
@nadezdatsygankova well done! Checked ✅

**Review by AnnaSolovykh — APPROVED** [review]

**Pull request merged**